### DISCUSSION TITLE
First blogs

### DISCUSSION THREAD

**Discussion by avinashupadhya99** [EddieHubCommunity/support#1038] [Show and tell]
Heyy! I recently wrote my first ever blog on Azure Functions. Here's a link to that - [My first experiment with Azure Functions](https://avinashupadhya99.medium.com/my-first-experiment-with-azure-functions-137b13b7b597).

**Comment by eddiejaoude** [comment]
Awesome 👍 

Here is a snippet for people

> I learnt serverless Azure Functions, CosmosDB and Azure DevOps through this challenge. I decided to take this learning further and implement a small application to solidify my understanding.

**Comment by aminoxix** [comment]
All about Open Source, quick start, & Opportunities : https://amino19.medium.com/open-source-opportunities-20b23db9e5a
A very first blog of mine 🚀

**Reply by verma-kunal** [reply]
Thats really an informative article! Kudos to the hard work 👍

**Comment by ashwinexe** [comment]
Hey everyone, this is my first blog also reflecting my journey since I joined developer communities. Hope you like it :)
I'm new to blogging so any review would be simply great!

https://ashwinexe.medium.com/opportunities-in-your-pocket-b121a0b81e2d

Here's a teaser :

> Just 2 months after I joined few developer communities online, I had participated in 10 hackathons, made 3 projects (2 full stack) and won 2 hackathons

**Reply by avinashupadhya99** [reply]
That's a really cool article. Short and covers everything in a fun way. Good one!

**Reply by ashwinexe** [reply]
Thanks Avinash! would try to write more.

**Comment by aminoxix** [comment]
Meanwhile Second blog : https://amino19.medium.com/career-in-postman-61efad9d0a52    ...xD
